supportedBackends/google/pipelines/v2beta/src/main/scala/cromwell/backend/google/pipelines/v2beta/api/Localization.scala

```diff
@@ -31,18 +36,43 @@ trait Localization {
       cloudPath = createPipelineParameters.cloudCallRoot / GcsDelocalizationScriptName,
       containerPath = gcsDelocalizationContainerPath))(mounts = mounts, labels = localizationLabel)
 
+    val drsLocalizationManifestContainerPath = createPipelineParameters.commandScriptContainerPath.sibling(DrsLocalizationManifestName)
+    val localizeDrsLocalizationManifest = cloudSdkShellAction(localizeFile(
+      cloudPath = createPipelineParameters.cloudCallRoot / DrsLocalizationManifestName,
+      containerPath = drsLocalizationManifestContainerPath))(mounts = mounts, labels = localizationLabel)
+
     val runGcsLocalizationScript = cloudSdkShellAction(
       s"/bin/bash $gcsLocalizationContainerPath")(mounts = mounts, labels = localizationLabel)
 
+    val runDrsLocalization = drsAction(createPipelineParameters, drsLocalizationManifestContainerPath, mounts, localizationLabel)
+
     // Any "classic" PAPI v2 one-at-a-time localizations for non-GCS inputs.
     val singletonLocalizations = createPipelineParameters.inputOutputParameters.fileInputParameters.flatMap(_.toActions(mounts).toList)
 
     val localizations =
       localizeGcsTransferLibrary ::
         localizeGcsLocalizationScript :: runGcsLocalizationScript ::
+        localizeDrsLocalizationManifest :: runDrsLocalization ::
         localizeGcsDelocalizationScript ::
         singletonLocalizations
 
     ActionBuilder.annotateTimestampedActions("localization", Value.Localization)(localizations)
   }
+
+  private def drsAction(createPipelineParameters: CreatePipelineParameters, manifestPath: Path, mounts: List[Mount], labels: Map[String, String]) = {
+    // TODO: Is this an acceptable way to read this config?
+    val config = ConfigFactory.load
+    val marthaConfig = config.getConfig("filesystems.drs.global.config.martha")
+    val drsConfig = DrsConfig.fromConfig(marthaConfig)
+    val drsDockerImage = config.getString("drs.localization.docker-image")
```

Comment:
I think we're losing information about requester pays project here - before we were getting it from the individual paths. Maybe we can include it as a third field in the manifest? I don't suppose we have any reason to believe all the DRS files localized for a particular task will have the same requester pays project id.

Reply:
I think we always use the workspace project as the requester-pays project, at least in Terra... which is what we want to focus on anyway, so we probably don't need a requester-pays project in the manifest.